appinventor/appengine/src/com/google/appinventor/server/GallerySearchIndex.java

```diff
@@ -5,30 +5,17 @@
 // http://www.apache.org/licenses/LICENSE-2.0
 
 package com.google.appinventor.server;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-
-import com.google.appengine.api.search.DeleteException;
-import com.google.appengine.api.search.Document;
-import com.google.appengine.api.search.Field;
-import com.google.appengine.api.search.Index;
-import com.google.appengine.api.search.IndexSpec;
-import com.google.appengine.api.search.PutException;
-import com.google.appengine.api.search.Query;
-import com.google.appengine.api.search.QueryOptions;
-import com.google.appengine.api.search.Results;
-import com.google.appengine.api.search.ScoredDocument;
-import com.google.appengine.api.search.SearchException;
-import com.google.appengine.api.search.SearchServiceFactory;
-import com.google.appengine.api.search.StatusCode;
+import com.google.appengine.api.search.*;
```

Comment:
@toropeza 

Bad: We don't like imports of the form .*; I suspect your IDE did this. Please revert this change and alter the settings of your IDE to not do this.